proxy/src/bin/proxy.rs

```diff
@@ -79,6 +80,9 @@ struct ProxyCliArgs {
     /// Allow self-signed certificates for compute nodes (for testing)
     #[clap(long, default_value_t = false, value_parser = clap::builder::BoolishValueParser::new(), action = clap::ArgAction::Set)]
     allow_self_signed_compute: bool,
+    /// timeout for http connections
+    #[clap(long, default_value_t = 15)]
+    sql_over_http_timeout: u64,
```

Comment:
I wonder if clap can accept a duration type here, eg `"15s"` and parse it

Reply:
Good idea! That works!